File: cpp/27a_1692A_Marathon.cpp
#include<bits/stdc++.h>

int main(){
    int t, a, b, c, d, cnt;
    std::vector<int> count;
    std::cin>>t;
    for(int i = 0; i < t; i++){
        cnt = 0;
        std::cin>>a>>b>>c>>d;
        if(b > a){
            cnt++;
        }
        if(c > a){
            cnt++;
        }
        if(d > a){
            cnt++;
        }
        count.push_back(cnt);
    }
    for(auto i : count){
        std::cout<<i<<"\n";
    }
    return 0;
}

File: python/27a_1692A_Marathon.py
def main():
    count = []
    t = int(input())
    for _ in range(t):
        cnt = 0
        a, b, c, d = input().split()
        a, b, c, d = int(a), int(b), int(c), int(d)
        if b > a:
            cnt += 1
        if c > a:
            cnt += 1
        if d > a:
            cnt += 1
        count.append(cnt)
    for i in count:
        print(i)

if __name__ == "__main__":
    main()